src/Products.jl

```diff
@@ -207,6 +207,86 @@ function locate(lp::LibraryProduct, prefix::Prefix; platform::Platform = platfor
     return nothing
 end
 
+"""
+A `FrameworkProduct` is a  [`Product`](@ref) that encapsulates a macOS Framework.
```

Comment:
Maybe you want to mention this product also in the docstring of `Product`?

Reply:
Done!